src/main/java/teammates/storage/api/CoursesDb.java

```diff
@@ -148,7 +149,7 @@ private Course getCourseEntity(String courseId) {
     protected CourseAttributes makeAttributes(Course entity) {
         Assumption.assertNotNull(Const.StatusCodes.DBLEVEL_NULL_INPUT, entity);
 
-        return CourseAttributes.builder(entity.getUniqueId(), entity.getName(), entity.getTimeZone())
+        return CourseAttributes.builder(entity.getUniqueId(), entity.getName(), ZoneId.of(entity.getTimeZone()))
```

Comment:
Actually, maybe do it in `Course` using `@OnLoad`, since it's only needed temporarily. After data migration it can be removed.

Reply:
Data is already using valid timezone string, so there's no need for migration? Anyway, I think it should be done here no matter what, in case the app fails somewhere and invalid timezone strings appear in the database.

Reply:
I was referring to `GMT+0`, `GMT-0` etc. that were supported by Joda time but not java.time. But you have a point - doing it here is good for fault tolerance. Let's do it here then.